packages/react-vapor/src/components/modalWizard/ModalWizard.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import {connect} from 'react-redux';
+
+import {ConnectedProps, IDispatch, TooltipPlacement} from '../../utils';
+import {Button} from '../button';
+import {ModalActions} from '../modal/ModalActions';
+import {IModalCompositeOwnProps, ModalCompositeConnected} from '../modal/ModalComposite';
+import {UnsavedChangesModalProvider} from '../modal/UnsavedChangesModalProvider';
+import {StepProgressBar} from '../stepProgressBar';
+
+export interface ModalWizardProps
+    extends Omit<IModalCompositeOwnProps, 'modalBodyChildren' | 'validateShouldNavigate'> {
+    id: string;
+    onFinish?: () => unknown;
+    validateStep?: (currentStep: number) => string[];
```

Comment:
I'm a bit saddened that it does not support dirty and warnings, reflecting the Validations feature :sweat_smile: 

I understand that the dirty is not quite needed since they can be translated into an error...

But the warnings are a nice feature to give some feedback: "You can go on, but X might not work because Y"

Reply:
I changed the signature to

```ts
validateStep?: (currentStep: number, isLastStep?: boolean) => {isValid: boolean; message?: string};
```